build/pipelines/templates/ci.yml

```
@@ -62,7 +62,7 @@ jobs:
       Write-Host $testsArray -separator "`n"
 
       # building tests will automatically build the main project as well
-      .\run-tests.ps1 -tests $testsArray
+      .\run-tests.ps1 -tests $testsArray -Configuration $env:Configuration
```

You can also perform variable substitution with ADO syntax if you want:
```suggestion
      .\run-tests.ps1 -tests $testsArray -Configuration $(Configuration)
```

---

Fixed -- I've updated the script to use ADO syntax.